### ISSUE TITLE
How to get autosize to work with hint?

### ISSUE THREAD

**Issue by LEMUSADR000** [lzhuor/auto_size_text_field#20]
Auto size doesn't appear to be working with the hint field. Is this not the expected use case?

**Comment by lzhuor** [comment]
Hi @LEMUSADR000 it is expected. Do you mind sharing more about what would you like to have for the hint field?

**Comment by LEMUSADR000** [comment]
@lzhuor yeah I was wanting a text field which had a hint that scaled up and down with increasing size depending on the content. If you enter a long string it would expect it to scale down to the min size allowed (user defined) and then fallback on the strategy of choice (user defined).

**Comment by berk-can** [comment]
what I did to workaround this issue is wrapping this with stack and putting hinttext on top showing when focusNode doesnt have focus
